src/screens/Home/components/ChooseLocation/components/MapLabel.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { Box, Button } from 'grommet'
+import styled from 'styled-components'
+import { func, string } from 'prop-types'
+
+export const StyledButton = styled(Button)`
+  padding: 0.5em;
+  white-space: nowrap;
+
+  :hover {
+    background: white;
+    text-decoration: underline;
+  }
```

Comment:
What's the focus styling?

Reply:
Whoops. Forgot that, and also got focus and blur working with showing a map on the homepage.